robotiq_modbus_rtu/src/robotiq_modbus_rtu/com_modbus_rtu.py

```diff
@@ -57,6 +56,7 @@ def __init__(self):
         self.read_registers_ac = None
         self.write_registers_ac = None
         self.modbus_action_lock = Lock()
+        self.robotiq_id = rospy.get_param("/grippers/robotiq_id", 9)
```

Comment:
Is there a corresponding PR to add this to the configuration?

Reply:
I assumed we didn't currently expect this to be anything but 9, but if that's incorrect, then I second @adamcalhoon 's question.

Reply:
As stated in the description, the teachmate node sets this parameter for now (it originally housed the ID as a constant).

I don't know if this paradigm is going to stay around long-term, so I don't want to put it in the configuration at this time.